xcp_d/cli/parser.py

```diff
@@ -79,7 +86,9 @@ def _build_parser():
             'The mode of operation for XCP-D. '
             'The mode sets several parameters, with values specific to different pipelines. '
             'For more information, see the documentation at '
-            'https://xcp-d.readthedocs.io/en/latest/workflows.html#modes'
+            'https://xcp-d.readthedocs.io/en/latest/workflows.html#modes. '
+            'If you are unsure which mode to use, we recommend using "none" mode and specifying '
```

Comment:
Are you sure we want to recommend this? It seems like a bad idea

Reply:
That's fair. Changing it now.